Tests/StyleguideTests/__Snapshots__/StyleguideTests/testDesignSystem.1.pretty.css

```diff
@@ -216,13 +216,13 @@
 }
 
 .col-m-1 {
-  flex-basis : 8.33333333333333%;
-  max-width  : 8.33333333333333%;
+  flex-basis : 8.333333333333334%;
```

Comment:
oh ha you can see these changes much more clearly here. 

I remember seeing some chatter about the floating point printing becoming more accurate in swift 4.2

Reply:
tho... this seems definitely NOT more accurate.